src/governance.rst

```diff
@@ -0,0 +1,172 @@
+Governance
+==========
+This document outlines the policies and procedures that manage the conda-forge
+community. This document recognizes that, while packaging is deeply important
+to all of us, conda-forge is an all-volunteer organization. Members of the
+core, any team, or external contributors choose to participate and may
+choose to leave at any time for any reason or for no reason. We deeply
+appreciate all good faith contributions.
+
+Code of Conduct
+---------------
+Conda-forge adheres to the
+`Numfocus Code of Conduct <https://www.numfocus.org/code-of-conduct>`_.
```

Comment:
Should we link to the versioned copy of the Code of Conduct or note the version?

ref: https://www.contributor-covenant.org/version/1/2/0/code-of-conduct.html

Reply:
It's worth noting there is a newer copy of this Code of Conduct, which we should compare the merits of.

ref: https://www.contributor-covenant.org/version/1/4/code-of-conduct

Reply:
I think the "latest version" should be the default -- I presume that that link goes to the latest version. If NumFocus happens to update it in a way we don't agree with at some arbitrary point in the future, we can revisit then.